manifests/j/JinweiZhiguang/MasterAgent/0.4.3.0/JinweiZhiguang.MasterAgent.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=QUSU.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: JinweiZhiguang.MasterAgent
+PackageVersion: 0.4.3.0
+PackageLocale: en-US
+Publisher: Beijing Jinwei Zhiguang Information Technology
+PublisherUrl: https://mastergo.com/
+PublisherSupportUrl: https://mastergo.com/help/
+PrivacyUrl: https://mastergo.com/privacyPolicy
+Author: Beijing Jinwei Zhiguang Information Technology Co., Ltd.
+PackageName: MasterAgent
+PackageUrl: https://mastergo.com/resource
+License: Freeware
+LicenseUrl: https://mastergo.com/serviceAgreement
+Copyright: Beijing Jinwei Zhiguang Information Technology
+# CopyrightUrl:
+ShortDescription: Create with fonts from your computer in MasterGo
```

```suggestion
ShortDescription: Professional UI/UX design tool for teams 
```
I think this is the correct description as mentioned in the package website, the font tool is a plugin I believe.